.circleci/config.yml

```diff
@@ -1195,6 +1195,7 @@ jobs:
     steps:
       - checkout
       - react-get-deps
+      - run: make -C webui check
```

Comment:
as is these are synonymous. is the idea that it'll be better if we just check the higher-level module and let that trickle down to `react` ?

Reply:
no, it's that I didn't realize they were synonymous.